docs/api-ref.md

```diff
@@ -1244,6 +1294,56 @@ None. The user is removed from the group.
 <br>
 <br>
 
+#### groups.remove_users
+
+```py
+groups.remove_users(group_item, user_id):
+```
+
+Removes multiple users to the specified group.
+
+
+REST API [Remove User from Group](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#remove_user_to_group)
+
+**Version**
+
+This endpoint is available with REST API version 3.21 and up. Method removeed in
```

Comment:
I think you meant "Method added in..." here

Reply:
Fixed.